vespa/test_integration_vespa_cloud.py

```diff
@@ -57,15 +57,15 @@ def setUp(self) -> None:
         )
         self.disk_folder = os.path.join(os.getenv("WORK_DIR"), "sample_application")
         self.instance_name = "".join(
-            random.choice(string.ascii_lowercase) for _ in range(24)
+            random.choice(string.ascii_lowercase) for _ in range(10)
```

Comment:
You can use `random.sample(string.ascii_lowercase, 10)` instead of the for loop.

Reply:
`random.sample` sample without replacement.